bundle/manifests/nonadmindownloadrequest-admin-role_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I am not sure we need these files;
- nonadmindownloadrequest-admin-role_rbac.authorization.k8s.io_v1_clusterrole.yaml
- nonadmindownloadrequest-editor-role_rbac.authorization.k8s.io_v1_clusterrole.yaml
- nonadmindownloadrequest-viewer-role_rbac.authorization.k8s.io_v1_clusterrole.yaml

Reply:
copies over from nac repo which is from `make manifests` I think.

Reply:
When installing bundle, these roles will be avail for admin to set rolebinding to users authorized to view or edit these downloadrequests

Reply:
Won't this give perms to get `nonadmindownloadrequests` across entire cluster to a non-admin user? 

Also we don't have those RBAC for other components, so we should be consistent here and either create bugs to add those for other objects (backups, restores, etc...) or not include this here, as I don't see reason why it is different?

Reply:
The role would look like this, right ?:

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
  name: nonadmindownloadrequest-reader-binding
  namespace: non-admin-restricted-namespace
subjects:
- kind: User
  name: non-admin-user
  apiGroup: rbac.authorization.k8s.io
roleRef:
  kind: ClusterRole
  name: nonadmindownloadrequest-admin-role
  apiGroup: rbac.authorization.k8s.io
```

Reply:
> We don't have those RBAC for other components, so we should be consistent here and either create bugs to add those for other objects (backups, restores, etc...) or not include this here, as I don't see reason why it is different?

+1 @mpryc

Reply:
ok so this came in via https://github.com/kubernetes-sigs/kubebuilder/commit/aa754c7a76e9b669ca278246c9d97c326a5aa4b3 in kubebuidler v4.5.1

Reply:
keep them I think, we just need to migrate others to 4.5.1 - my idea of merging this quickly /

Reply:
Did we resolve this? Was this because of the repeated kubebuilder markers ?

Reply:
> Did we resolve this?

>> keep them I think

I think yes?

> Was this because of the repeated kubebuilder markers ?

No, it came from kubebuilder v4.5.1. I have removed the duplicate markers in nac pr.

Reply:
> we just need to migrate others to 4.5.1

After 1.5 or before?